docs.phtml

```diff
@@ -712,7 +712,7 @@ $older7		="3";
 </ul>
 
 <div class="dialog note">
-<p class="justify"><?php echo T_("<b>NOTE:</b> the information presented in these sources may or may not be updated to the most recent version of <span class=\"gnucash\">GnuCash</span>. If you are interested in doing development work on these documents, you should contact the original authors.");?></p>
+<p class="justify"><?php echo T_("<b>Note:</b> Info presented in these sources may or may not be align with the latest <span class=\"gnucash\">GnuCash</span>. Contact the authors if you are interested in improving these resources.");?></p>
 </div>
```

Comment:
What about:

Note: The information presented in these resources may or may not be updated to the most recent version of GnuCash. Contact the original authors if you are interested in improving them.

Reply:
It is from, respectively July 2001 and November 4, 2002. I think it is time to let go here.